### PR TITLE
Release/1.0

### PR DESCRIPTION
**Pull request by MarcoOquendoC**
# [JavaScript group capstone: day 3](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)

### APIs
- Find an API to base the development of the webapp around it.
- Use [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions (likes, comments and reservations).

### Interfaces
- Build these interfaces:
  - The home page.
  - The comments popup.

### Counters 
- The number of items (home).
- The number of comments (comments popup).

### Technical set up
- Set up the repository on GitHub and use Gitflow.
- Set up webpack.
- Set up a JavaScript testing library (Jest).

## Project documentation
- A description of the project.
- A demo of the project features.

Patricio greeting

### CODE REVIEW

File: src/module/mealApi.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const getMeals = async (url) => {
+  const response = await fetch(url).then((response) => response.json());
+  // console.log(response.result);
+  return response;
```
Comment (codepantha) [inline src/module/mealApi.js]: - [x] Please remove all commented code as it makes your work look untidy 👍

File: src/module/modal.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import commentApi from './commentApi.js';
+
+const modalMethods = {
+  async show(id, src, mealName, category) {
+    // const meals = await getMeals(url);
+
```
Comment (codepantha) [inline src/module/modal.js]: - [x] Please remove all commented code as it makes your work look untidy 👍

File: src/module/modal.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    const modal = document.querySelector('#staticBackdrop');
+    modal.innerHTML = '';
+    const allContent = document.createElement('div');
+    allContent.classList.add('modal-dialog');
+    allContent.innerHTML = `
+      <div class="modal-content">
+        <div class="modal-header">
+          <h2 class="modal-title" id="staticBackdropLabel">${mealName}</h2>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
```
Comment (codepantha) [inline src/module/modal.js]: - [x] When the comments popup loads, the web app should retrieve data from the selected API and should show details about the selected item. At the moment, you are only retrieving data from the Involvement to show the item comments

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 🟢 

## Hi team 👋, 

Awesome job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Used GitFlow
✔️ Beautiful and responsive UI design


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MarcoOquendoC** [comment]
# Hi @codepantha,

Thanks for your kindness. The issues are corrected. 😅

little girl passing keys

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**